frontends/main/src/app-pages/DashboardPage/DashboardLayout.tsx

```diff
@@ -77,28 +77,14 @@ const Background = styled.div(({ theme }) => ({
   },
 }))
 
-const Page = styled.div(({ theme }) => ({
-  display: "flex",
-  flexDirection: "column",
-  alignItems: "flex-start",
-  padding: "40px 84px 80px 84px",
-  gap: "80px",
-  height: "100%",
-  [theme.breakpoints.down("md")]: {
-    padding: "0",
-    gap: "24px",
-  },
-}))
-
-const DashboardContainer = styled(Container)(({ theme }) => ({
-  [theme.breakpoints.down("md")]: {
-    padding: "24px 16px",
-    gap: "24px",
-  },
-}))
+const PageContainer = styled(Container)({
```

Comment:
Main change in this PR is getting rid of `DashboardContainer`, and using the builtin margins/padding on `Container` itself.